contracts/engine/GenArt721MinterDAExp_PBAB.sol

```diff
@@ -0,0 +1,611 @@
+                (success_, ) = genArtCoreContract.renderProviderAddress().call{
+                    value: renderProviderAmount
+                }("");
+                require(success_, "Renderer payment failed");
+                remainingFunds -= renderProviderAmount;
+            }
+
+            // Owner (platform provider) payment
+            uint256 ownerFunds = (remainingFunds * ownerPercentage) / 100;
+            if (ownerFunds > 0) {
+                (success_, ) = ownerAddress.call{value: ownerFunds}("");
+                require(success_, "Owner payment failed");
+                remainingFunds -= ownerFunds;
+            }
+
+            // Artist additional payee payment
+            uint256 additionalPayeePercentage = genArtCoreContract
+                .projectIdToAdditionalPayeePercentage(_projectId);
+            if (additionalPayeePercentage > 0) {
+                uint256 additionalPayeeAmount = (remainingFunds *
+                    additionalPayeePercentage) / 100;
+                if (additionalPayeeAmount > 0) {
+                    (success_, ) = genArtCoreContract
+                        .projectIdToAdditionalPayee(_projectId)
+                        .call{value: additionalPayeeAmount}("");
+                    require(success_, "Additional payment failed");
+                    remainingFunds -= additionalPayeeAmount;
+                }
+            }
+
+            // Artist payment
+            if (remainingFunds > 0) {
+                (success_, ) = genArtCoreContract
+                    .projectIdToArtistAddress(_projectId)
+                    .call{value: remainingFunds}("");
+                require(success_, "Artist payment failed");
+            }
+        }
+    }
+
+    /**
+     * @notice Gets price of minting a token on project `_projectId` given
+     * the project's AuctionParameters and current block timestamp.
+     * Reverts if auction has not yet started or auction is unconfigured.
+     * @param _projectId Project ID to get price of token for.
+     * @return current price of token in Wei
+     * @dev This method calculates price decay using a linear interpolation
+     * of exponential decay based on the artist-provided half-life for price
+     * decay, `_priceDecayHalfLifeSeconds`.
+     */
+    function _getPrice(uint256 _projectId) private view returns (uint256) {
+        ProjectConfig storage _projectConfig = projectConfig[_projectId];
+        // move parameters to memory if used more than once
+        uint256 _timestampStart = uint256(_projectConfig.timestampStart);
+        uint256 _priceDecayHalfLifeSeconds = uint256(
+            _projectConfig.priceDecayHalfLifeSeconds
+        );
+        uint256 _basePrice = _projectConfig.basePrice;
+
+        require(block.timestamp > _timestampStart, "Auction not yet started");
+        require(_priceDecayHalfLifeSeconds > 0, "Only configured auctions");
+        uint256 decayedPrice = _projectConfig.startPrice;
```

Comment:
if we think this minter might be ~commonly used as a "fixed price" minter via setting start price == end price, I would recommend a short-circuit to avoid some math (ga$) by
```solidity
// pseudo-code
if (startPrice == basePrice) {
    return basePrice;
}
// else calc price as usual
...
```

Reply:
Good call, saves a lil gas and is also nice and readable (and will be covered by the updated tests noted above). Updating now.

Reply:
Actually... it appears that this increases gas costs when we do:

```
        // Return early if configured in "fixed price mode" for gas efficiency
        if (_projectConfig.startPrice == _projectConfig.basePrice) {
            return _projectConfig.basePrice;
        }
``` 

While it is slightly more idiomatic, I think it is fine to just let this fall through as it would otherwise, but PLMK if I'm overlooking here.

<img width="779" alt="Screenshot 2023-01-19 at 6 48 46 PM" src="https://user-images.githubusercontent.com/8602661/213600727-d9647df6-4fdd-47c6-baea-3b0fddb980db.png">

Reply:
```
// Return early if configured in "fixed price mode" for gas efficiency
        if (_basePrice == _decayedPrice) {
            return _projectConfig.basePrice;
        }
```

Reply:
(might be better names we could use 😅)

Reply:
All set! :) Done!